about.html

```diff
@@ -0,0 +1,146 @@
+    <nav class="navbar">
+        <div class="container-logo">
+            <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <a href="/" class="logo">CC Global Summit 2015</a>
+        <div id="myNav" class="list-layout">
+            <div class="closeBtn-menu">
+                <a href="" class="closeBtn" onclick="closeNav()">&times;</a>
+            </div>
+            <ul class="nav-list">
+                <li><a href="/" class="list-link">Home</a></li>
+                <li><a href="about.html" class="list-link">About</a></li>
+                <li><a href="#" class="list-link">Program</a></li>
+                <li><a href="#" class="list-link">Join</a></li>
+                <li><a href="#" class="list-link">Speaker</a></li>
+                <li><a href="#" class="list-link">News</a></li>
+                <li><a href="#" class="list-link">CC Campaign</a></li>
+            </ul>
+        </div>
+    </nav>
+</header>
+   <div class="container">
+       <div class="small-title">
+           <h4 class="subtitle">"Hello! HIV Awareness"</h4>
+       </div>
+       <div class="big-title">
+           <h1 class="page-title">HIV AND AIDS AWARENESS  2022</h1>
+       </div>
+
+       <div class="about-para">
+           <p>HIV is a virus that targets and alters the immune system, increasing the risk and impact of other infections and diseases.
+           </p>
+       </div>
+       <div class="contact-us">
+           <p>
+               Please send written correspondence to:
+
+               HIV.gov / Office of Infectious Disease and HIV/AIDS Policy (OIDP)
+               U.S. Department of Health and Human Services
+           </p>
+       </div>
+    <div class="sub-sections">
+        <h2>HIV Global Summit 2022</h2>
+        <p>People living with HIV, family members and caregivers can access a variety of resources through Illinois HIV Care Connect.</p>
+    </div>
+    <div class="logo-heading">
+        <div class="image-logo">
+            <img src="assets/images/hiv.png" class="hiv" alt="">
+        </div>
+        <div class="sub-main">
+            <h3>HIV And Global Summit 2022 </h3>
+            <p>15 - 17th oct 2022 | Kampala, Uganda</p>
+        </div>
+    </div>
+    <div class="container-2">
+        <h3> Let's see the past HIV Global Summit</h3>
+        <div class="center-hor">
+            <div class="horizontal-line"></div>
+        </div>
+        <p>Regional trends in the annual number of new HIV infections and the prevalence density among all ages has reduced.</p>
+    </div>
+    <div class="big-container">
+        <div class="image-container">
+            <div class="background-images">
+                <h3>2022</h3>
+                <p>HIV Flash back 2022 in Uganda</p>
+            </div>
+            <div class="background-images">
+                <h3>2020</h3>
+                <p>HIV Flash back 2020 in Uganda</p>
+            </div>
+        </div>
+    </div>
+   </div>
+</div>
```

Comment:
- [ ] [optional] I am marking these change as optional because it was missed by the previous reviewer but I strongly suggest considering them. Please apply the color effect to the images.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169515291-fa60787e-a858-4562-858b-2d593fb40b61.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169515344-ef3522ad-be30-4274-8297-4279ba47fa93.png)